### PR TITLE
EMBCESSMOD-5004 - Billeting Option disabled

### PR DESCRIPTION
**Pull request by TimaQT**
EMBCESSMOD-5004 - Lodging - Billeting Option disabled, Extend and Edit blocked for existing Supports of this type

### CODE REVIEW

File: responders/src/UI/embc-responder/src/app/feature-components/wizard/support-components/existing-support-details/existing-support-details.component.html [hunk 1]
```diff
@@ -297,7 +297,7 @@
             </div>
           </div>
 
-          <div class="row">
+          <div class="row" *ngIf="canCloneSupport()">
```
Comment (ytqsl) [inline responders/src/UI/embc-responder/src/app/feature-components/wizard/support-components/existing-support-details/existing-support-details.component.html:300]: did you mean canEditSupport?

File: responders/src/UI/embc-responder/src/app/feature-components/wizard/support-components/select-support/select-support.component.ts [hunk 2]
```diff
@@ -31,7 +31,7 @@ export class SelectSupportComponent implements OnInit {
   ngOnInit(): void {
     this.supportList = this.loadEvacueeListService
       .getSupportTypeList()
-      .filter((element) => element.description !== '');
+      .filter((element) => element.description !== '' && element.value !== 'Lodging_Billeting');
```
Comment (ytqsl) [inline responders/src/UI/embc-responder/src/app/feature-components/wizard/support-components/select-support/select-support.component.ts:34]: should this compare against the enum value and not text?

File: responders/src/UI/embc-responder/src/app/feature-components/wizard/support-components/existing-support-details/existing-support-details.component.html (hunk 1 above)
Comment (TimaQT) [reply responders/src/UI/embc-responder/src/app/feature-components/wizard/support-components/existing-support-details/existing-support-details.component.html:300]: I've created 2 function - canEdit and canClone support as in future they could have different conditions. 
CanEdit is in this code as well, but checks only Edit button. And canClone - is a condition for  3 'Clone & Extend' buttons

File: responders/src/UI/embc-responder/src/app/feature-components/wizard/support-components/select-support/select-support.component.ts (hunk 2 above)
Comment (TimaQT) [reply responders/src/UI/embc-responder/src/app/feature-components/wizard/support-components/select-support/select-support.component.ts:34]: it is pretty common that we compare against text  ( which is a value), so I would create a separate story to replace all the values with enum option values

File: responders/src/UI/embc-responder/src/app/feature-components/wizard/support-components/existing-support-details/existing-support-details.component.html (hunk 1 above)
Comment (ytqsl) [reply responders/src/UI/embc-responder/src/app/feature-components/wizard/support-components/existing-support-details/existing-support-details.component.html:300]: canCloneOrExtend? Probably better separate the functions and still have the same logic in them for a future maintainer

File: responders/src/UI/embc-responder/src/app/feature-components/wizard/support-components/select-support/select-support.component.ts (hunk 2 above)
Comment (ytqsl) [reply responders/src/UI/embc-responder/src/app/feature-components/wizard/support-components/select-support/select-support.component.ts:34]: No story for this - just fix it as you see them please. Here is a good start.

File: responders/src/UI/embc-responder/src/app/feature-components/wizard/support-components/existing-support-details/existing-support-details.component.html (hunk 1 above)
Comment (TimaQT) [reply responders/src/UI/embc-responder/src/app/feature-components/wizard/support-components/existing-support-details/existing-support-details.component.html:300]: created 2 functions, and used both.

File: responders/src/UI/embc-responder/src/app/feature-components/wizard/support-components/select-support/select-support.component.ts (hunk 2 above)
Comment (TimaQT) [reply responders/src/UI/embc-responder/src/app/feature-components/wizard/support-components/select-support/select-support.component.ts]: fixed

**Review by ytqsl — APPROVED** [review]

**Pull request merged**